proto/internal.proto

```diff
@@ -40,4 +41,23 @@ message UpdateCacheRequest {
 
 // Response of UpdateCache request.
 message UpdateCacheResponse {
-}
\ No newline at end of file
+}
+
+// Request to get data in the import table
+message GetImportTableDataRequest {
+}
+
+// Response of GetImportTableDataRequest
+message GetImportTableDataResponse{
+  message ImportData {
+    message ImportMetadata{
+      int32 num_obs = 5;
```

Comment:
To accommodate unset values, consider making them `optional`.

Reply:
Good idea, made them `optional`.